ios/browser/api/brave_rewards/brave_rewards_api.mm

```diff
@@ -42,6 +44,26 @@
 #error "This file requires ARC support."
 #endif
 
+namespace {
+
+using brave_rewards::RemoteWorker;
+
+class HashPrefixStoreWorker
+    : public RemoteWorker<brave_rewards::mojom::HashPrefixStore> {
+ public:
+  HashPrefixStoreWorker() : RemoteWorker(CreateTaskRunner()) {}
+  ~HashPrefixStoreWorker() = default;
+
+ private:
+  static scoped_refptr<base::SequencedTaskRunner> CreateTaskRunner() {
+    return base::ThreadPool::CreateSequencedTaskRunner(
+        {base::MayBlock(), base::TaskPriority::USER_VISIBLE,
+         base::TaskShutdownBehavior::BLOCK_SHUTDOWN});
```

Comment:
maybe `SKIP_SHUTDOWN`?
Also does it make sense to reuse one of the current task runner to simplify things?